mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -675,9 +675,22 @@ def find_resource(self, res_name, res_dirname=None):
         Returns:
             string: The full path to the resource file or None if not found
         """
+        result = self._find_resource(res_name, self.lang, res_dirname)
+        if not result:
```

Comment:
Can you add `and self.lang != 'en-us'` to this if-statement so it doesn't try to get English resources twice if the file is missing.